packages/php-wasm/logger/src/lib/handlers/log-to-memory.ts

```diff
@@ -0,0 +1,67 @@
+import { LogHandler } from '../log-handlers';
+import { LogSeverity, Log } from '../logger';
+
+const formatLogDate = (date: Date): string => {
+	const formattedDate = new Intl.DateTimeFormat('en-GB', {
+		year: 'numeric',
+		month: 'short',
+		day: '2-digit',
+		timeZone: 'UTC',
+	})
+		.format(date)
+		.replace(/ /g, '-');
+
+	const formattedTime = new Intl.DateTimeFormat('en-GB', {
+		hour: '2-digit',
+		minute: '2-digit',
+		second: '2-digit',
+		hour12: false,
+		timeZone: 'UTC',
+		timeZoneName: 'short',
+	}).format(date);
+	return formattedDate + ' ' + formattedTime;
+};
+
+const prepareLogMessage = (message: any, ...args: any[]): string => {
+	return [
+		typeof message === 'object' ? JSON.stringify(message) : message,
+		...args.map((arg) => JSON.stringify(arg)),
+	].join(' ');
+};
+
+const formatLogEntry = (
```

Comment:
I don't want to overcomplicate this PR, but it feels like many handlers could want the same format. Perhaps making formatter a property of the logger, not log handler, would make more sense?

Reply:
Done https://github.com/WordPress/wordpress-playground/pull/1264/commits/c9f90281fcb83bd8f4f6054d7fa402e84c864ff7.